client/notebooks/Demo_Client_Notebook_Smartnoise-SQL.ipynb

```diff
@@ -800,415 +622,401 @@
     }
    ],
    "source": [
-    "client.get_remaining_budget()"
+    "dummy_std = np.round(dummy_res['query_response']['std_bill_length_mm'].iloc[0], 2)\n",
+    "f\"The dummy standard variation is {dummy_std}.\""
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 19,
-   "id": "69767fac",
+   "cell_type": "markdown",
+   "id": "74f68994",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "avg_bill_length_response = client.smartnoise_sql_query(\n",
-    "    query = QUERY,  \n",
-    "    epsilon = EPSILON, \n",
-    "    delta = DELTA,\n",
-    "    dummy = False\n",
-    ")"
+    "The syntax of the query works, now she checks the budget:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "id": "6dbbdf93",
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Average bill length: 44.18mm.\n"
-     ]
-    }
-   ],
-   "source": [
-    "avg_bill_length = avg_bill_length_response['query_response']['avg_bill_length_mm'].iloc[0]\n",
-    "print(f\"Average bill length: {np.round(avg_bill_length, 2)}mm.\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "b2767e65",
+   "execution_count": 19,
+   "id": "a8fa2c49",
    "metadata": {},
+   "outputs": [],
    "source": [
-    "After each query on the real dataset, the budget informations are also returned to the researcher. It is possible possible to check the remaining budget again afterwards:"
+    "cost = client.estimate_smartnoise_sql_cost(\n",
+    "    query = QUERY, \n",
+    "    epsilon = 0.5, \n",
+    "    delta = 1e-5\n",
+    ")"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
-   "id": "39701fe5",
+   "execution_count": 20,
+   "id": "b3aa05ca-3243-4415-a8ec-fb5ad47d244d",
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "{'remaining_epsilon': 6.885714285713345,\n",
-       " 'remaining_delta': 0.004994999999999967}"
+       "'This query would actually cost her 1.5 epsilon and 5.000000000032756e-06 delta.'"
       ]
      },
-     "execution_count": 21,
+     "execution_count": 20,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "client.get_remaining_budget()"
+    "f'This query would actually cost her {cost[\"epsilon_cost\"]} epsilon and {cost[\"delta_cost\"]} delta.'"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "e37c587f",
+   "id": "884f0337-a960-460e-8797-84ddd77974a3",
+   "metadata": {},
+   "source": [
+    "This times it is three times the budget as the standard deviation needs the average, then a difference and a count again. "
```

Comment:
This time (without the S)
This time it is three times the budget because the standard ...